appcues/src/main/java/com/appcues/ui/theme/AppcuesTheme.kt

```diff
@@ -50,9 +52,9 @@ private fun getColors(isDark: Boolean): Colors {
  * This is supposed to be used during inspection mode (Preview) only.
  */
 @Composable
-internal fun AppcuesPreview(
+internal fun AppcuesPreviewPrimitive(
```

Comment:
changing the Preview to read the primitive as ExperiencePrimitive and call ExperiencePrimitive.Compose in order to apply all modifiers properly.